servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/AuthorizationRuleSerializer.java

```diff
@@ -149,6 +150,8 @@ private static AuthorizationRule parseSasAuthRule(Element xEntry) {
                         }
                         rule.setRights(rights);
                         break;
+                    default:
```

Comment:
[SpotBugs-P2]
This method contains a switch statement where default case is missing. Usually you need to provide a default case.